src/app/technical-record/technical-record.component.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from '@app/models/vehicle-tech-record.model';
 import { TestResultModel } from '@app/models/test-result.model';
 import { VIEW_STATE } from '@app/app.enums';
+import { DialogMinistryPlatesComponent } from '@app/shared/dialog-ministry-plates/dialog-ministry-plates.component';

```

Comment:
move under this import `import { AdrReasonModalComponent } from '@app/shared/adr-reason-modal/adr-reason-modal.component';`